src/react-redux/provider.js

```diff
@@ -0,0 +1,42 @@
+import { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+
+class FreshDataReduxProvider extends Component {
+	static propTypes = {
+		children: PropTypes.node.isRequired,
+		rootPath: PropTypes.oneOfType( [
+			PropTypes.arrayOf( PropTypes.oneOfType( [ PropTypes.string, PropTypes.number ] ) ),
+			PropTypes.string,
+		] ),
+		update: PropTypes.func,
+	};
+
+	static defaultProps = {
+		rootPath: [ 'freshData' ],
+		update: () => {}, // TODO: Hook in to registry module by default.
+	};
+
+	componentDidMount() {
+		const { rootData, update } = this.props;
```

Comment:
Yeah, I thought about that too. Since they're basically one-liners right now, I'll not create another function for it. But if this gains more logic, I'll add an intermediate function for sure. Thanks!